env/mock_env.cc

```diff
@@ -379,7 +379,8 @@ class TestMemLogger : public Logger {
       gettimeofday(&now_tv, nullptr);
       const time_t seconds = now_tv.tv_sec;
       struct tm t;
-      localtime_r(&seconds, &t);
+      auto ret __attribute__((__unused__)) = localtime_r(&seconds, &t);
```

Comment:
Oh interesting, didn't know! Seems like we have both the versions in our code base.